nemo/collections/common/metrics/classification_accuracy.py

```diff
@@ -59,6 +59,8 @@ def validation_epoch_end(self, outputs):
         accuracy, compute acc=correct_count/total_count
     """
 
+    full_state_update = True  # https://devblog.pytorchlightning.ai/torchmetrics-v0-9-faster-forward-d595bb321e6d
```

Comment:
```suggestion
    full_state_update = True
```
I'd remove the comment primarily since the base class should explain what this does, putting it everywhere would be mundane!

Reply:
yes, haven't commented anywhere else. Will remove it here as well

Reply:
done it already!